src/view/Controllers/AnimalQueriesController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using PetHome.Handlers.Queries;
+using viewmodels;
+
+namespace view.Controllers
+{
+    [Route("animals")]
+    [ApiController]
+    public class AnimalQueriesController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public AnimalQueriesController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+        [HttpGet]
```

Comment:
after #20 we might want to add the attributes to this api endpoint, before final merge